SPECS/maven-surefire/maven-surefire.spec

```diff
@@ -0,0 +1,515 @@
+%global upstream_version %(echo '%{version}' | tr '~' '-')
+%bcond_without bootstrap
+Summary:        Test framework project
+Name:           maven-surefire
+Version:        3.0.0
+Release:        1%{?dist}
+License:        ASL 2.0 AND CPL
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://maven.apache.org/surefire/
+# ./generate-tarball.sh
+Source0:        %{_mariner_sources_url}/%{name}-%{version}.tar.gz
+# Remove bundled binaries which cannot be easily verified for licensing
+Source1:        generate-tarball.sh
```

Comment:
this file does not seem to be used

Reply:
generate-tarball.sh is used to strip jar files from the source.

Reply:
removed as not needed to be embedded